### PR TITLE
Add Data Center GPU Flex Series to Readme

### PR DESCRIPTION
**Pull request by yangw1234**
## Description

Add Data Center GPU Flex Series to Readme

### CODE REVIEW

File: python/llm/example/transformers/transformers_int4/GPU/baichuan/README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # Baichuan
-In this directory, you will find examples on how you could apply BigDL-LLM INT4 optimizations on Baichuan models on any Intel® Arc™ A-Series Graphics. For illustration purposes, we utilize the [baichuan-inc/Baichuan-13B-Chat](https://huggingface.co/baichuan-inc/Baichuan-13B-Chat) as a reference Baichuan model.
+In this directory, you will find examples on how you could apply BigDL-LLM INT4 optimizations on Baichuan models on any Intel® GPUs. For illustration purposes, we utilize the [baichuan-inc/Baichuan-13B-Chat](https://huggingface.co/baichuan-inc/Baichuan-13B-Chat) as a reference Baichuan model.
```
Comment (jason-dai) [inline python/llm/example/transformers/transformers_int4/GPU/baichuan/README.md:2]: In this directory, you will find examples on how you could apply BigDL-LLM INT4 optimizations on Baichuan models on any [Intel GPUs](../README.md).

Comment (yangw1234) [reply python/llm/example/transformers/transformers_int4/GPU/baichuan/README.md]: fixed. 

And I guess it is not accurate to say "any Intel GPUs", so I also remove the "any".

**Comment by jason-dai** [comment]
I think you can remove all the trademark signs; otherwise LGTM

**Comment by cyita** [comment]
Please update the starcoder readme.

**Comment by yangw1234** [comment]
> I think you can remove all the trademark signs; otherwise LGTM

fixed

**Comment by yangw1234** [comment]
> Please update the starcoder readme.

updated

**Comment by rnwang04** [comment]
shall we change `Intel GPU` to `Intel dGPU` as we do't support iGPU now.

**Comment by jason-dai** [comment]
> shall we change `Intel GPU` to `Intel dGPU` as we do't support iGPU now.

We cam just keep GPU

**Comment by jason-dai** [comment]
> shall we change `Intel GPU` to `Intel dGPU` as we do't support iGPU now.

We can keep GPU for now

**Review by songhappy — APPROVED** [review]

**Pull request merged**